Extending/Section-Trees/trees-v7.md

```diff
@@ -95,7 +95,7 @@ eg:
 
 ### Responding to Tree Actions
 
-The actions on items in an Umbraco Tree will trigger 'by convention' a request to load an AngularJS view, with a name corresponding to the name of the action, from a subfolder of the views folder matching the name of the 'customTreeAlias'.
+The actions on items in an Umbraco Tree will trigger 'by convention' a request to load an AngularJS view, with a name corresponding to the name of the action, from a sub-folder of the views folder matching the name of the 'customTreeAlias'.
```

Comment:
A subfolder is allowed by https://en.oxforddictionaries.com/definition/subfolder

Reply:
Reverted :)